src/code/z_gameover.c

```diff
@@ -0,0 +1,151 @@
+#include "global.h"
+
+void GameOver_Init(GlobalContext* globalCtx) {
+    globalCtx->gameOverCtx.state = GAMEOVER_INACTIVE;
+}
+
+void GameOver_FadeInLights(GlobalContext* globalCtx) {
+    GameOverContext* gameOverCtx = &globalCtx->gameOverCtx;
+
+    if ((gameOverCtx->state >= GAMEOVER_DEATH_WAIT_GROUND && gameOverCtx->state < GAMEOVER_REVIVE_START) ||
+        (gameOverCtx->state >= GAMEOVER_REVIVE_RUMBLE && gameOverCtx->state < GAMEOVER_REVIVE_FADE_OUT)) {
+        Kankyo_FadeInGameOverLights(globalCtx);
+    }
+}
+
+// This file has the same issue as z_message where data is going to rodata.
+// This hack allows the function to OK, though rodata still needs to be fixed along with z_message rodata.
+extern const s16 D_80153D80[];
+#define NON_CONST(x, type) ((type*)(&x))
+#define GAME_OVER_TIMER NON_CONST(D_80153D80, s16)[0]
```

Comment:
Because of how I ended up implementing the fix on the message branch, I believe it would be fine to remove this hack now if it still matches.
Try changing the extern to simply `extern s16 D_80153D80;` and replacing all `GAME_OVER_TIMER` with `D_80153D80` directly as if it were a regular data variable. Hopefully it still matches.
If this works you can move the extern to `variables.h` as well.

Reply:
it does match, yeah. Thanks.
think ill leave the extern in the file tho, as the variable isnt global and doesnt need to be accessible elsewhere